pkg/api/secretbootstrap/secretboostrap_test.go

```diff
@@ -209,6 +217,41 @@ func TestLoadConfigFromFile(t *testing.T) {
 	}
 }
 
+func TestRoundtripConfig(t *testing.T) {
+	testCases := []struct {
+		name string
+	}{
+		{
+			name: "basic base",
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			c := &Config{}
+			inFile := filepath.Join("testdata", fmt.Sprintf("%s.yaml", t.Name()))
```

Comment:
Use `testhelper.ReadFromFixture()`

Reply:
Correct me if I'm wrong, but I think I actually want to use the `secretbootstrap.LoadConfigFromFile` function here to get all the custom Unmarshaling. I could get the same result using `testhelper.ReadFromFixture()` and following that up with `yaml.Unmarshal()` but that seems like more effort

Reply:
What you get for free with `testhelper.ReadFromFixture()` is the test data file name management - it has utility in being central and common. Calling `yaml.Unmarshal` is the expected cost of Go's verbosity.